dsa-23au/DSAFormat.md

```diff
@@ -0,0 +1,231 @@
+# Steppenwolfenstein: a Data Structures and Algorithms Format 
+
+We define a serialization / deserialization format for recording
+data structures and algorithms, both as a snapshot and as an execution
+trace as they progress from an input to an output.
+
+This language, and the engine that runs it, is known as Steppenwolfenstein 
+abbreviated (SWS).
+
+It steps through DSA and allows them to come to glorious blocky, pixelated life.
+Also, you can enjoy it to the classic 1968 song "Magic Carpet Ride" popularized
+in the film Star Trek: Generations, performed by Steppenwolf.
+
+SWS is only a display language. It is not a complete representation of the 
```

Comment:
What are the advantages to a "display language" verses a standard language?

Reply:
This is a term I made up. If you need exact data results, you would use a standard language system that runs on a server. If you're okay with a vague picture of the data, the standard language system would send display language updates to (for example) your web browser or phone, but it may arrive late, be inconsistent between frames, or not have high-resolution if you want to zoom in.